_pages/platform/organization-profile.md

```diff
@@ -71,12 +71,12 @@ Once members of your organization have created an algorithm and submitted it for
 
 **Note:** Every new version of the algorithm to be published will require independent approval.
 
-At this time you'll be able to set the royalty (if any) if you're using our non-Enterprise platform, and finalize publishing.
+At this time you'll be able to set the royalty (if any) if you're using our non-Enterprise platform and finalize publishing.
```

Comment:
Thanks; everything after this point actually applies to our deprecated public marketplace platform and will be going away, but I can make those changes myself.